pkg/runctl/run_manager.go

```diff
@@ -577,6 +571,17 @@ func getTimeout(runCtx *runContext) *metav1.Duration {
 	return timeout
 }
 
+func getTokenTimeout(runCtx *runContext) int64 {
+	timeoutSeconds := defaultTokenTimeoutSeconds
```

Comment:
Using a default token lifetime that is independent from the TaskRun execution timeout does not make sense, because it can lead to the situation that a pipeline still runs but the token is invalid (-> infrastucture problem).

Instead, we should introduce an own default timeout for TaskRun executions that is used if the pipeline runs config does not provide one.